.utils/deploy.sh

```diff
@@ -0,0 +1,119 @@
+#!/bin/bash
+
+# This file is used to deploy yttest to an S3 bucket. It
+# expects the S3 bucket name in an environment variable
+# $YTTEST_BUCKET
+
+set -ex
+
+main() {
+  # For short-lived assets; in seconds
+  FIVE_DAYS="432000"
+
+  # For long-lived assets; in seconds
+  ONE_YEAR="31536000"
+
+  CSPSTATIC="\"content-security-policy\": \"default-src 'none'; "\
+"base-uri 'none'; "\
+"form-action 'none'; "\
+"object-src 'none'\""
```

Comment:
This is great! If the static content isn't framed anywhere I'd add `frame-ancestors: 'none'` too since it doesn't fallback to `default-src`. 

https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/frame-ancestors

Reply:
Oh also if we have a version of Sentry with Fx CSP report support we could add a report-uri pointing there.

Reply:
`frame-ancestors: 'none'` added.

> if we have a version of Sentry with Fx CSP report support we could add a report-uri pointing there.

We don't have that yet.